src/Orleans.Core/Core/DefaultClientServices.cs

```diff
@@ -34,6 +34,7 @@ public static void AddDefaultServices(IClientBuilder builder, IServiceCollection
             services.TryAddFromExisting<IInternalGrainFactory, GrainFactory>();
             services.TryAddFromExisting<IGrainReferenceConverter, GrainFactory>();
             services.TryAddSingleton<ClientProviderRuntime>();
+            services.TryAddSingleton<IFieldUtils>(new FieldUtils());
```

Comment:
any reason why this is passing an instance instead of just the concrete type, as you did in `DefaultSiloServices`?

Reply:
Nope, will fix that